### PR TITLE
Update Editorconfig

### PR DESCRIPTION
**Pull request by hanzei**
If you are fine with this, I would apply this style to all files.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -5,8 +5,17 @@ root = true
...
 indent_style = space
 indent_size = 4
+
+[*.toml]
+indent_style = space
+indent_size = 2
+
+[*.go]
+indent_style = tab
+indent_size = 2
```
Comment (hmhealey) [inline .editorconfig]: Is the indent_size needed here? We don't have it defined in the server's .editorconfig, so I don't know if it does anything here.

Comment (hanzei) [reply .editorconfig]: BTH it might not be needed there. But it does no harm here.

**Comment by hanzei** [comment]
@hmhealey I would like to apply this style to all files. Are you fine with this?

File: .editorconfig (hunk 1 above)
Comment (hmhealey) [reply .editorconfig]: That's fine then. I'll just add Joram as a second reviewer then so we can make sure to get two reviews before merging this

**Pull request merged**